api/src/pcapi/connectors/titelive.py

```diff
@@ -6,55 +6,70 @@
 from pcapi import settings
 from pcapi.core import logging as core_logging
 from pcapi.utils import requests
+from pcapi.utils.cache import get_from_cache
 
 
 logger = logging.getLogger(__name__)
 
 
-def get_jwt() -> str:
+def get_jwt_token() -> str:
+    TITELIVE_JWT_CACHE_KEY = "api:titelive_jwt:cache"
+    TITELIVE_JWT_CACHE_TIMEOUT = 5 * 60  # 5 minutes
```

Comment:
Micro-détail : je me demande si on ne devrait pas cacher pendant _légèrement moins_ que 5 minutes.

En indiquant 5 minutes, avec beaucoup (mais alors beaucoup !) de malchance, on pourrait récupérer un token à l'instant T + 0 (donc expirant côté API à T + 300), le stocker dans Redis à l'instant T + 1 (donc expirant dans Redis à l'instant T + 1 + 300, le récupérer de Redis à l'instant T + 300, et l'envoyer dans la requête HTTP à l'API à l'instant T + 300 + 1, et se faire jeter par l'API car le token n'est plus valide.

C'est peu probable parce que les temps de latence HTTP et Redis sont courts et qu'il faudrait que les appels soient faits dans des intervalles extrêmement serrés, donc t'as le droit d'ignorer mon blabla. :) Sinon : 

```suggestion
    TITELIVE_JWT_CACHE_TIMEOUT = 5 * 60 - 5  # slightly less than the lifetime of the token (5 minutes)
```

Reply:
En faites, tout dépend, est-ce que le timer compte depuis le début du retriever ou a la fin, si c'est a la fin, et j'y ai pensé je suis d'accord, mais je serais d'avis de laisser comme ça, et de corriger si on rencontre des problèmes. Qu'en dis-tu ?